slack/client.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	historicalRequestLimit = 500
-	batchSendInterval      = 900 * time.Second // 15 minutes
+	batchSendInterval      = 30 * time.Second // 15 minutes
```

Comment:
Seems like the value was left from testing.
Wdyt about making this configurable via command-line flag?
That should make it less error-prone when testing as it will be possible to use lower value when running tests.